ui/site/src/site.ts

```diff
@@ -74,6 +74,15 @@ lichess.load.then(() => {
       }, 200);
     });
 
+    if (window.matchMedia('(max-width: 650px)').matches) {
+      $('table.slist').each(function (this: HTMLTableElement) {
+        const container = document.createElement('div');
+        container.style.overflowX = 'auto';
+        container.style.maxWidth = '100vw';
+        this.replaceWith(container);
+        container.appendChild(this);
+      });
+    }
```

Comment:
I would prefer to add the wrapper `div.slist-wrapper` on the serverside, to prevent FOUC and executing extra JS.
We can also use that wrapper to explicitely define where we want the horizontal scrolling.

Reply:
Sure, site.ts is becoming quite the patchwork quilt of random hacks as well.